test/llvm-ir-tests.js

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2012-2018, Patrick Quist
```

Comment:
Please update the copyright here.

Thank you _so_ much for adding a test here! It really helps us!